### PR TITLE
Add update_field_value action

### PR DESCRIPTION
**Pull request by ytjohn**
As described in #35, creates an action to update_field_value where you can set an existing field to a new value.

### CODE REVIEW

File: actions/update_field_value.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    description: the field text itself.
+    required: true
+  notify:
+    type: boolean
+    description: jira will send notifications (default is true)
+    default: true
+    required: false
+
+
+
```
Comment (blag) [inline actions/update_field_value.yaml]: Can you remove the two extra newlines here? The file should end with only one newline.

**Review by blag — CHANGES_REQUESTED** [review]

**Comment by blag** [comment]
Looks good, aside from the tiny YAML file lint issue I noted.

CI is failing, but it's not due to anything in this PR. I'll rerun the CI jobs for this PR once you fix the tiny linting thing and once https://github.com/StackStorm/st2/pull/4819 is merged.

**Comment by ytjohn** [comment]
Updated with corrections.

**Review by blag — CHANGES_REQUESTED** [review]
Now that I got the CI jobs actually running, the flake8 linting section of the CI jobs identified two issues. I've converted them into GitHub suggestions here.

Sorry to string you along like this, but you should now have accurate feedback from the CI checks for this PR. Once you fix the remaining linting issues and CI checks all pass, I'd be happy to merge it. :smile:

**Comment by ytjohn** [comment]
No problem. As the saying goes, a poor craftsman blames his tools. In this case, it was VSCode's fault as it likes to insert blank lines with spaces as a way of being "helpful". :smile:

**Review by blag — APPROVED** [review]
Thanks for fixing all of the linting issues!

**Comment by blag** [comment]
Thanks for the contribution!

**Pull request merged**